core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/PhysicalPlan.scala

```diff
@@ -322,17 +322,6 @@ case class PhysicalPlan(
     chains.filter(s1 => chains.forall(s2 => s1 == s2 || !s1.subsetOf(s2))).toSet
   }
 
-  def setOperatorUnblockPort(
-      physicalOpId: PhysicalOpIdentity,
-      portIdToRemove: PortIdentity
-  ): PhysicalPlan = {
-    val physicalOp = getOperator(physicalOpId)
-    physicalOp.copy(blockingInputs =
-      physicalOp.blockingInputs.filter(port => port != portIdToRemove)
-    )
-    this.copy(operators = operators ++ Set(physicalOp))
-  }
-
   def areAllInputBlocking(physicalOpId: PhysicalOpIdentity): Boolean = {
```

Comment:
this function still uses "blocking input concept". can you update/remove it?

Reply:
removed